inc/admin/class-cenote-notice.php

```diff
@@ -0,0 +1,93 @@
+<?php
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+class Cenote_Notice {
+	public $name;
+	public $type;
+	public $dismiss_url;
+	public $temporary_dismiss_url;
+	public $pricing_url;
+	public $current_user_id;
+
+	/**
+	 * The constructor.
+	 *
+	 * @param string $name Notice Name.
+	 * @param string $type Notice type.
+	 * @param string $dismiss_url Notice permanent dismiss URL.
+	 * @param string $temporary_dismiss_url Notice temporary dismiss URL.
+	 *
+	 * @since 1.3.2
```

Comment:
This class will be added in next update 1.3.3 so, it should be the same

Reply:
I have made the changes